### PR TITLE
app-vim/*: Bump EAPI 5 -> 7

### PR DESCRIPTION
**Pull request by mjsir911**
Signed-off-by: Marco Sirabella <marco@sirabella.org>

CC @monsieurp 

Prerequisite for #20530

### CODE REVIEW

File: app-vim/screen/screen-1.5.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```
Comment (telans) [inline app-vim/screen/screen-1.5.ebuild:4]: 6?

File: app-vim/vim-addon-mw-utils/vim-addon-mw-utils-20121105.ebuild [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```
Comment (telans) [inline app-vim/vim-addon-mw-utils/vim-addon-mw-utils-20121105.ebuild:4]: likewise

File: app-vim/screen/screen-1.5.ebuild (hunk 1 above)
Comment (mjsir911) [reply app-vim/screen/screen-1.5.ebuild:4]: oh, thanks! my mistake

**Comment by mjsir911** [comment]
Actually I think I'm going to bundle all these commits in with #20530

**Pull request closed without merging**